charts/sourcegraph/values.yaml

```diff
@@ -661,6 +661,56 @@ openTelemetry:
       requests:
         cpu: "100m"
         memory: 100Mi
+
+nodeExporter:
+  # -- Enable `node-exporter`
+  enabled: true
+  image:
+    # -- Docker image tag for the `node-exporter` image
+    defaultTag: 179720_2022-10-25_4d925e87cfb8@sha256:2d9dcdf0b2226f0c3d550a64d2667710265462350a3ba9ebe37d0302bc64af0f
```

Comment:
I used a valid image tab, but this isn't one that comes from a release (since haven't cut a release since node-exporter was introduced in https://github.com/sourcegraph/sourcegraph/pull/43107. Will the release tooling handle this special tag?

Reply:
Seems fine either way. We should be able to check post-release and update then if needed.